migration/process-migration/src/main/java/io/camunda/migration/process/MigrationRunner.java

```diff
@@ -126,6 +148,14 @@ private boolean isImporterFinished() {
     }
   }
 
+  private double migratedProcessesCount(
+      final String lastMigratedProcessDefinitionKey, final List<ProcessEntity> processes) {
+    return processes.stream()
+            .takeWhile(process -> !process.getId().equals(lastMigratedProcessDefinitionKey))
```

Comment:
🤔 ❓  Do we need this? This list contains only not migrated processes right?

Reply:
No the List contains all entities that were supposed to be migrated. If the `lastMigratedProcessDefinitionKey` is different than the last element of the list, the count will not be same as the `processes.size`. 

Could be possible to refactor this a bit and simply return the migrated entities, will check it.

Reply:
I will go ahead and merge this. We can iterate from there.

Reply:
BTW does it mean we have always a list of process definitions in memory? 🤔 Might be quite huge or?

Reply:
It should be equal to `BATCH_SIZE` on each migration round apart from the last that can be less.